nuxeo-core-binarymanager-common/src/test/java/org/nuxeo/ecm/blob/AbstractTestCloudBinaryManager.java

```diff
@@ -83,7 +83,7 @@ protected void removeObject(String digest) throws IOException {
 
     @Before
     public void setUp() throws IOException {
-        assumeTrue("Cannot set Unlimited JCE Policy", setUnlimitedJCEPolicy());
+        //assumeTrue("Cannot set Unlimited JCE Policy", setUnlimitedJCEPolicy());
```

Comment:
cause:

```
/**
     * By default the JRE may ship with restricted key length. Instead of having administrators download the Java
     * Cryptography Extension (JCE) Unlimited Strength Jurisdiction Policy Files from
     * http://www.oracle.com/technetwork/java/javase/downloads/index.html, we attempt to directly unrestrict the JCE
     * using reflection.
     * <p>
     * This is not possible anymore since 8u102 and https://bugs.openjdk.java.net/browse/JDK-8149417
     */
```

But I can re-enable it if you want